### PR TITLE
Version Change for BitZeny

### PR DESCRIPTION
**Pull request by hitobb**
This is a request to change the version rule to BitZeny from BitCoin 15.1 (Base Source).
I want to make this version to 2.0.0.

### CODE REVIEW

**Review by onokatio — APPROVED** [review]
LGTM

File: configure.ac [hunk 1]
```diff
@@ -164,7 +164,7 @@ AC_ARG_ENABLE([lcov],
   [enable lcov testing (default is no)])],
   [use_lcov=yes],
   [use_lcov=no])
-
```
Comment (ghost) [inline configure.ac]: Please remove unnecessary changes

File: configure.ac [hunk 2]
```diff
@@ -1286,7 +1286,7 @@ case ${OS} in
    ;;
 esac
 
-echo
```
Comment (ghost) [inline configure.ac]: same

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by hitobb** [comment]
Fix line feed code CRLF to LF
Please check again.

**Review by ghost — APPROVED** [review]
LGTM

**Comment by hitobb** [comment]
Merged
